teuthology/orchestra/run.py

```diff
@@ -240,24 +240,24 @@ def __eq__(self, value):
         return self.value == value
 
 
-def quote(args):
+def convert_args_list_to_str(args):
     """
-    Internal quote wrapper.
+    Convert list args to str args, in case of Raw instances use their value
+    for generating str arg.
     """
-    def _quote(args):
-        """
-        Handle quoted string, testing for raw charaters.
-        """
-        for a in args:
-            if isinstance(a, Raw):
-                yield a.value
-            else:
-                yield pipes.quote(a)
-    if isinstance(args, list):
-        return ' '.join(_quote(args))
-    else:
+    if isinstance(args, str):
         return args
 
+    assert isinstance(args, list)
+
+    for a in args:
+        if isinstance(a, Raw):
+            a = a.value
+        else:
+            a = pipes.quote(a)
+
+    return ' '.join(_quote(args))
```

Comment:
where is `_quote()` defined ?

Reply:
I was supposed to remove the call to `_quote()` here. Thanks for pointing out!